src/server/graphql/types/GitHubMemberRemovedPayload.js

```diff
@@ -0,0 +1,13 @@
+import {GraphQLList, GraphQLObjectType} from 'graphql';
+import LeaveIntegrationPayload from 'server/graphql/types/LeaveIntegrationPayload';
+
+const GitHubMemberRemovedPayload = new GraphQLObjectType({
+  name: 'GitHubMemberRemovedPayload',
+  fields: () => ({
+    leaveIntegration: {
```

Comment:
in the future, this payload will also include a notification, but for now, notifications are still handled by cashay & the rehthinkdb livequery